### ISSUE TITLE
OpenFaas integration

### ISSUE THREAD

**Issue by alexdebrie** [serverless/event-gateway#452]

**Comment by alexellis** [comment]
Let is know how we can help @alexdebrie

**Comment by alexdebrie** [comment]
Thanks, @alexellis!

What's the best way to consume an OpenFaas function -- an HTTP endpoint, or is there something like the [`Invoke`](https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html) API call for AWS Lambda?

**Comment by alexellis** [comment]
Yes this is in our [documentation](https://docs.openfaas.com/) and the [Swagger definitions](https://github.com/openfaas/faas/blob/master/api-docs/swagger.yml).

**Comment by alexellis** [comment]
Bump
